### DISCUSSION TITLE
need some help with a double realy

### DISCUSSION THREAD

**Discussion by aihysp** [arendst/Tasmota#12805] [Q&A] [answered]
first i want to say i am "noob" i have nu clue in electronics

i have two relays in tasmota that control one valve , when both ,are off
the COM out ,of two of them will give 12V+(Open)
when both are on it will put 12V-(Closed)
when one of them is off and the other is on it will give out zero volts

what can i do to make it more easy for me to turn it on and off with one switch ?
i want to turn it on for lets say 10 secs and then cut off power with some kind rule?
can two relays be combined to one?

**Comment by Pewibe** [comment, marked as answer]
I didn't understand your hint about electronics and your next chapter of text because there is no question included. The last chapter of text has questions which have nothing to do with voltages.
Q1: what can i do to make it more easy for me to turn it on and off with one switch ? A: Use rules!
Q2: i want to turn it on for lets say 10 secs and then cut off power with some kind rule? A: Rule example:
```
Rule1 
    ON Switch1#State=0 DO Backlog Power1 OFF; Power2 ON; RuleTimer1 0; ENDON
    ON Switch1#State=1 DO Backlog Power1 ON; Power2 OFF; RuleTimer1 100; ENDON
    ON Rules#Timer=1 DO Backlog Power1 OFF; Power2 ON; RuleTimer1 0; ENDON
Rule1 ON; 
```
Q3: can two relays be combined to one? A: Read in https://tasmota.github.io/docs/Commands/#control about 'interlock'

Generally you shoud read the docs in total to answer your questions by yourself.

**Reply by aihysp** [reply]
first i want to say thanks!! 
i am a long tasmota user, and reading allot of docs, my first idea was to use shutter ...
so again thank you i will try this!!!

**Comment by rapi3** [comment]
Hi,
It is possible to combine 2 or more relays with: Interlock
It is possible to have your relay to stay ON or OFf for a defined time with: PulseTime
have a look at documentation: [https://tasmota.github.io/docs/Commands/](url)
